logstash-core/spec/api/spec_helper.rb

```diff
@@ -92,7 +94,7 @@ def do_request(&block)
     @runner = LogStashRunner.new
     @runner.start
   end
-
+  
```

Comment:
the whitespace things here just bring the whitespace to the correct point, indented with blacks.